src/com/oltpbenchmark/schema/defaultschema/DefaultTable.java

```diff
@@ -0,0 +1,48 @@
+package com.oltpbenchmark.schema.defaultschema;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.oltpbenchmark.schema.Column;
+import com.oltpbenchmark.schema.Table;
+import com.oltpbenchmark.schema.TableSchema;
+
+public class DefaultTable extends Table {
+    
+    private final String tablespace;
+    
+    public DefaultTable(TableSchema schema) {
+        super(schema);
+        tablespace = null;
+    }
+    
+    public DefaultTable(TableSchema schema, String tablespace) {
+        super(schema);
+        this.tablespace = tablespace;
+    }
+
+    @Override
+    public String getCreateDdl() {
+        StringBuilder sb = new StringBuilder();
+        sb.append(String.format("CREATE TABLE %s ( ", schema.name()));
+        boolean is_first = true;
```

Comment:
It is even better to avoid using format when StringBuilder is used

```
for (Column c : schema.getColumns()) {
   sb.append('\n').append(c.getName()).append(' ').append(c.getDecl()).append(',');
}
sb.setLength(sb.length() - 1);
```

Reply:
Nice points, done